src/MathOptInterface.jl

```diff
@@ -51,6 +51,13 @@ Read the file `filename` into the instance `instance`. If `m` is non-empty, this
 throw an error.
 
 Supported file types depend on the instance type.
+
+### Note
+
+Once the model is loaded into the model, users can query the variables via
+`get(model, ListOfVariableIndices())`. However, this could return the variables
+in any order. Therefore, users should look up up each variable index by name:
```

Comment:
MPS files have a well-defined order on the variables. Would we still expect `ListOfVariableIndices()` to return them back in an arbitrary order in that case?

Reply:
Presumably. But that seems a little flakey.

Reply:
This is another manual doc then? Recommending `get.(model, VariableIndex, ["x", "y", "z"])` without mentioning `ListOfVariableIndices`?